Map/Test/FieldTest.cpp

```diff
@@ -0,0 +1,12 @@
+#include "EmptyField.hpp"
+#include <gtest/gtest.h>
+
+using namespace ::testing;
+
+TEST(FieldTest, makePrintable_ShouldChangePrintability)
```

Comment:
camelCase

Reply:
Chciałem oddzielić nazwę metody od oczekiwanego zachowania

Reply:
rozumiem, ale nie jest to potrzebne. makePrintableShouldChangePrintability też tworzy ładną cała historyjkę. Dodatkowo jest to troche ogólna nazwa. Napisałeś że powinna zmienić, ale w jaki sposób ?